tools/synthgen/internal/synthgen/cli.go

```
@@ -28,18 +28,20 @@ const (
 // Run 执行 synthgen CLI（argv 不含程序名），返回进程退出码。
 func Run(argv []string, stdout, stderr io.Writer) int {
 	if len(argv) == 0 {
-		fmt.Fprintln(stderr, "usage: synthgen <register|generate|dist-check> [flags]")
+		fmt.Fprintln(stderr, "usage: synthgen <register|generate|generate-neg|dist-check> [flags]")
```

golangci-lint (2.12.2)

[error] Error return value of `fmt.Fprintln` is not checked

(errcheck)